### ISSUE TITLE
rinad does not build on pristine-1.4

### ISSUE THREAD

**Issue by vmaffione** [IRATI/stack#835]
````
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I/home/vmaffione/git/irati/rinad/src/common -I../../src -pthread -I/home/vmaffione/git/irati/local/include -I/usr/include/libnl3 -pthread -I/home/vmaffione/git/irati/rinad/src/common/tclap -g -O2 -MT librinad_la-configuration.lo -MD -MP -MF .deps/librinad_la-configuration.Tpo -c /home/vmaffione/git/irati/rinad/src/common/configuration.cc  -fPIC -DPIC -o .libs/librinad_la-configuration.o
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I/home/vmaffione/git/irati/rinad/src/common -I../../src -pthread -I/home/vmaffione/git/irati/local/include -I/usr/include/libnl3 -pthread -I/home/vmaffione/git/irati/rinad/src/common/tclap -g -O2 -MT librinad_la-configuration.lo -MD -MP -MF .deps/librinad_la-configuration.Tpo -c /home/vmaffione/git/irati/rinad/src/common/configuration.cc -o librinad_la-configuration.o >/dev/null 2>&1
mv -f .deps/librinad_la-configuration.Tpo .deps/librinad_la-configuration.Plo
/bin/sh ../../libtool  --tag=CXX   --mode=compile g++ -DHAVE_CONFIG_H -I. -I/home/vmaffione/git/irati/rinad/src/common -I../../src   -pthread -I/home/vmaffione/git/irati/local/include -I/usr/include/libnl3 -pthread -I/home/vmaffione/git/irati/rinad/src/common/tclap			   -g -O2 -MT librinad_la-encoder.lo -MD -MP -MF .deps/librinad_la-encoder.Tpo -c -o librinad_la-encoder.lo `test -f 'encoder.cc' || echo '/home/vmaffione/git/irati/rinad/src/common/'`encoder.cc
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I/home/vmaffione/git/irati/rinad/src/common -I../../src -pthread -I/home/vmaffione/git/irati/local/include -I/usr/include/libnl3 -pthread -I/home/vmaffione/git/irati/rinad/src/common/tclap -g -O2 -MT librinad_la-encoder.lo -MD -MP -MF .deps/librinad_la-encoder.Tpo -c /home/vmaffione/git/irati/rinad/src/common/encoder.cc  -fPIC -DPIC -o .libs/librinad_la-encoder.o
/home/vmaffione/git/irati/rinad/src/common/encoder.cc:36:33: fatal error: encoders/IntType.pb.h: No such file or directory
compilation terminated.
Makefile:618: recipe for target 'librinad_la-encoder.lo' failed
make[4]: *** [librinad_la-encoder.lo] Error 1
make[4]: Leaving directory '/home/vmaffione/git/irati/build/rinad/src/common'
Makefile:700: recipe for target 'install-recursive' failed
make[3]: *** [install-recursive] Error 1
make[3]: Leaving directory '/home/vmaffione/git/irati/build/rinad/src/common'
Makefile:406: recipe for target 'install-recursive' failed
make[2]: *** [install-recursive] Error 1
make[2]: Leaving directory '/home/vmaffione/git/irati/build/rinad/src'
Makefile:441: recipe for target 'install-recursive' failed
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory '/home/vmaffione/git/irati/build/rinad'
Makefile:738: recipe for target 'install' failed
make: *** [install] Error 2
install-user-from-scratch: Cannot complete rinad phase
````

**Comment by vmaffione** [comment]
Seems to be introduced by this commit

````
6e9f3c7fdfc1188c162d4f0db117c06b0e38f4bb
````

**Comment by vmaffione** [comment]
My first impression is that files are missing. @bernatgaston do you have some suggestions?

**Comment by vmaffione** [comment]
It seems to be related to #832

**Comment by bernatgaston** [comment]
I have just downloaded a clean version of pristine-1.4 compiled, and everthing went well. This is beacuase your makefile.inc in rinad/src/common/encoders still have the information about IntType.proto which was removed. I am looking at pristine-1.4 and everything is correct. Please check that your makefile.inc is the last version https://github.com/IRATI/stack/blob/pristine-1.4/rinad/src/common/encoders/Makefile.inc

**Comment by bernatgaston** [comment]
Or maybe try to configure again, seems a failing update of the makefile

**Comment by vmaffione** [comment]
Everything went well because I already fixed this, see #836  :)
Closing this
